packages/core/src/builderProvider/subscriber/index.ts

```diff
@@ -117,6 +117,10 @@ export function subscriber(event: MessageEvent): void {
       case `${target}_dc_image_rej`:
       case `${target}_dc_link_res`:
       case `${target}_dc_link_rej`:
+      case `${target}_dc_placeholder_data_res`:
```

Comment:
need to be like here
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/18303258/79aad890-d84c-4087-87e2-dfc232a4b8c7)